native_projects/snake/.webinizer/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "__type__": "ProjectConfig",
+  "name": "Snake",
+  "desc": "Snake is a classic video puzzle game, implemented based on SDL2.",
+  "img": {
+    "name": "snake.png",
+    "isUploaded": true
+  },
+  "category": "Game",
+  "id": 0,
```

Comment:
`"id": 0` is used for tetris demo. The value is used to determine the display order of demo projects in UI and should be unique. You may need to set another one.